src/data/apiClient.js

```diff
@@ -13,4 +16,69 @@ const apiClient = getAuthenticatedAPIClient({
   csrfCookieName: configuration.CSRF_COOKIE_NAME,
 });
 
+const clientServerKeyMap = {
+  bio: 'bio',
+  socialLinks: 'social_links',
+  country: 'country',
+  education: 'level_of_education',
+  fullName: 'name',
+  username: 'username',
+  profileImage: 'profile_image',
+  dateJoined: 'date_joined',
+  languageProficiencies: 'language_proficiencies',
+  accountPrivacy: 'account_privacy',
+};
+
+const serverClientKeyMap = _.invert(clientServerKeyMap);
```

Comment:
A map like this would eliminate our snakekeys and camelcase transformer dependencies and adds clarity.

Reply:
🐍 🐫 🐍 🐫 🐍 🐫 🐍 🐫 🐍 🐫 🐍 🐫 🐍 🐫 🐍 🐫 🐍 🐫